ui/seek_bar.js

```diff
@@ -110,6 +110,7 @@ shaka.ui.SeekBar = class extends shaka.ui.RangeElement {
         });
 
     // Initialize seek state and label.
+    this.wasPlaying_ = null;
```

Comment:
style: please declare this above the event listeners, please declare its type, and please keep the type consistent (just boolean, not null)

Ex:

```js
/** @private {boolean} */
this.wasPlaying_ = false;
```

Reply:
Done - I'm including your comment into source code as well, is it common in shaka-player to link to a github issue ? I'm happy to remove it though.